docs/src/agents.jl

```diff
@@ -60,14 +60,16 @@ fig
 
 # ## Interactive ABM Applications
 # Continuing from the Daisyworld plots above, we can turn them into interactive
-# interactive applications straightforwardly, simply by providing the stepping functions
+# applications straightforwardly, simply by providing the stepping functions
 # as illustrated in the documentation of [`abmplot`](@ref).
+# Note that `GLMakie` [should be used](https://discourse.julialang.org/t/the-vs-code-plot-navigator-and-makie/67967)
```

Comment:
This link points to something specific to VSCode, but GLMakie is not specific to VSCode. If you want to put a link, it can only be to the Makie website, where the state all their backends.

Reply:
> Note that [`GLMakie`](https://makie.juliaplots.org/v0.15/documentation/backends_and_output/) should be used instead of `CairoMakie` if on VSCode.

?

Reply:
Okay, now I see the problem. The original statement in the docstring is missleading. This issue has nothing to do with VSCode. GLMakie is a Makie backend that enables interactivity. Doesn't matter if you run Julia via VSCode or vim. The correct statement is "Note that [GLMakie] (link you used) should be used instead of `CairoMakie` when wanting to use the interactive aspects of the plots.". Can you please put this statement instead, and completely remove any reference to VSCode, that anyways shouldn't have been there from the start?

Reply:
Makes sense, didn't know it's not specific to code